cmd/agent/api/grpc.go

```diff
@@ -36,6 +36,7 @@ import (
 
 const (
 	taggerStreamSendTimeout = 1 * time.Minute
+	streamKeepAliveInterval = 9 * time.Minute
```

Comment:
I think it'd be good to have a comment somewhere explaining why sending an empty message every 9 min is necessary, basically what you explained in the PR description.

Reply:
I’ve just added a comment here: https://github.com/DataDog/datadog-agent/pull/11834/files#diff-f3315955be2dc5c5534b0b750fb43048ef236fa190db75e5272210ecf8fa3fd9R174-R178